compiler/test/fail_compilation/ice23279.d

```diff
@@ -0,0 +1,13 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/ice23279.d(12): Error: undefined identifier `Sth`
+---
+*/
+module ice23279;
+
+class Tester
+{
+    enum a = __traits(hasMember, Tester, "setIt");
+    void setIt(Sth sth){}
+}
```

Comment:
Just to be on the safe side, could you add a test with the __traits expression **after** the `setIt` method?